root/programs/C#/Frameworks/Infrastructure/ServiceInterface/ASPNETWebService/ASPNETWebService/App_Start/RouteConfig.cs

```diff
@@ -0,0 +1,46 @@
+﻿//**********************************************************************************
```

Comment:
@daisukenishino 

I think this project contains web services (ASP.NET web services, WCF, and Web API) only. Since this project does not contain screens (like .aspx, .cshtml), I think that this file is unnecessary.